packages/webapp/components/layouts/AccountLayout/index.tsx

```diff
@@ -107,6 +108,8 @@ export const getAccountLayout = (
   page: ReactNode,
   props: AccountLayoutProps,
 ): ReactNode =>
-  getMainLayout(<AccountLayout {...props}>{page}</AccountLayout>, null, {
-    screenCentered: false,
-  });
+  getFooterNavBarLayout(
+    getMainLayout(<AccountLayout {...props}>{page}</AccountLayout>, null, {
+      screenCentered: false,
+    }),
+  );
```

Comment:
By wrapping in the FooterNavBarLayout, we're able to have the profile menu open while navigating to account pages.
Found no side-effects of doing this.